sdk/typescript/src/types/events.ts

```diff
@@ -53,3 +54,38 @@ export type SuiEvent =
     | { newObject: NewObjectEvent }
     | { epochChange: bigint }
     | { checkpoint: bigint };
+
+export type MoveEventField = {
+    path: string,
+    value: SuiJsonValue
+}
+
+// mirrors sui_framework::EventType
+export type EventType =
+    | "TransferToAddress"
+    | "TransferToObject"
+    | "FreezeObject"
+    | "ShareObject"
+    | "DeleteObjectID"
+    | "User";
```

Comment:
This is not what `EventType` is referring to. It refers to https://github.com/MystenLabs/sui/blob/main/crates/sui-types/src/event.rs#L105 . It should be `"MoveEvent" | "Publish" |....`.


https://docs.rs/strum_macros/latest/strum_macros/derive.EnumDiscriminants.html